CONTRIBUTION.md

```diff
@@ -0,0 +1,161 @@
+
+### Attribution
+
+This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4,
+available at [http://contributor-covenant.org/version/1/4][version]
+
+[homepage]: http://contributor-covenant.org
+[version]: http://contributor-covenant.org/version/1/4/
+
+## Pull Request Process 
+
+1. Eclipse Che documentation project follows forking workflow. To learn more, read [Atlassian Git Tutorial, Forking Workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/forking-workflow). Before you start working, fork the repository, make sure you create all branches in your own fork.
+2. `master`  is the default working branch. Unless your change is intended for a specific version, send the PR against a `master`. See the Release branches to learn more.
+3. Specify as much information as possible in the PR template. 
+
+### Release branches
+[WIP]
+
+## Creating documentation
+
+### AsciiDoc Formatting
+Eclipse Che documentation use AsciiDoc for formatting. See [AsciiDoc Writer's Guide](https://asciidoctor.org/docs/asciidoc-writers-guide/) for syntax and general help with AsciiDoc.
+
+#### Using `test-adoc` for AsciiDoc syntax verification
+To verify that the syntax of your document is correct, use [`test-adoc`](https://github.com/jhradilek/check-links) script. 
+`test-adoc.sh` is integrated into the `run.sh` script in the root directory of the project.
+ 
+To run `test-adoc`, execute from the root directory of the project: 
+```
+$ bash run.sh -test-adoc <PATH_TO_THE_FILE>
+```
+
+### Documentation Structure
+Eclipse Che documentation follow [Modular Documentation initiative](https://redhat-documentation.github.io/modular-docs/). Use templates provided in the [Modular Documentation Reference](https://redhat-documentation.github.io/modular-docs/#creating-modules) to create new user stories.
+
+To add a new page, create an `.adoc` file in `src/main/pages/che-<version>/<directory_name>` 
+
+<b id="f2">Versions: </b>
+* `che-6` - Eclipse Che 6 content. Note that Che 6 is deprecated. 
+* `che-7` - Eclipse Che 7 content, including y-streams and z-streams.   
+
+<b id="f1">Available directories: </b> 
+* `overview` - introductory section
+* `end-user-guide` - documentation for developers: navigating dashboard, working in Che-Theia, and so on 
+* `installation-guide` - installation guides
+* `administration-guide` - documentation for administrators of the clusters: configuring Che on a cluster, managing users, monitoring resources, security and data recovery 
+* `contributor-guide` - how to develop plug-ins for Che, add new debuggers, languages, and so on
+* `extensions` - documentation about extensions for Che, such as Eclipse Che4z, OpenShift Connector 
+
+#### Using `newdoc` to create templates for new documents
+
+With a [`newdoc`](https://github.com/redhat-documentation/tools/tree/master/newdoc) script, you can generate empty modular templates for your new user story.  
```

Comment:
Is this "a `newdoc`" script or "the `newdoc`" script? I don't know how this is implemented, so I thought it was worth asking. :)